### ISSUE TITLE
Add load before functionality 

### ISSUE THREAD

**Issue by KANAjetzt** [GodotModding/godot-mod-loader#114]
>KANA — Yesterday at 16:46
 I need a way to tell a mod ( Darkly77-ContentLoader ) that it has to wait for my mod ( KANA-WhatAmILookingAt )

> Ste — Yesterday at 16:49
technically only by dependencies.. we could introduce a "load-before" to store more mod ids that should load after this mod
load before would essentially copy over the importance score of the other mod and add it to the own importance

**Comment by KANAjetzt** [comment]
Load before field in the *manifest.json* in *extra* -> *godot* -> *load_before*
`"load_before": ["Darkly77-ContentLoader"],`

```JSON
{
	"name": "WhatAmILookingAt",
	"namespace": "KANA",
	"version_number": "1.0.0",
	"description": "Tells you where this thing is from.",
	"website_url": "https://github.com/BrotatoMods",
	"dependencies": [],
	"extra": {
		"godot": {
			"incompatibilities": [],
			"load_before": ["Darkly77-ContentLoader"],
			"authors": ["KANA"],
			"compatible_mod_loader_version": "4.3.0",
			"compatible_game_version": ["0.8.0.3"],
			"config_defaults": {}
		}
	}
}
```

**Comment by KANAjetzt** [comment]
closed by #185
